src/config/default.js

```diff
@@ -222,6 +222,19 @@ export default {
           },
         }),
       },
+      exhibitionTitle: {
+        field: 'collectionspace_denorm:exhibition.title',
+        messages: defineMessages({
+          label: {
+            id: 'filter.title.label',
```

Comment:
The convention is to use the filter name in the message id, so this should be `filter.exhibitionTitle.label`.